dcos-log/api/v2/handlers.go

```diff
@@ -121,7 +156,7 @@ func filesAPIHandler(w http.ResponseWriter, req *http.Request) {
 		opts = append(opts, reader.OptLines(limit))
 	}
 
-	cursor := -1
+	var cursor = -1
```

Comment:
just curious, why did you change this to a `var` declaration?

Reply:
not sure, but good catch